<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputer</class>
 <widget class="QDialog" name="AddComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelAddComputerHeading">
     <property name="text">
      <string>&lt;h1&gt;Add Computer&lt;/h1&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelComputerName">
       <property name="text">
        <string>Model Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelModelNameError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditComputerName">
     <property name="placeholderText">
      <string>Enter model name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelComputerMakeYear">
       <property name="text">
        <string>Make year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMakeYearError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMakeYear">
     <property name="placeholderText">
      <string>Enter year computer was made</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelComputerType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxComputerType"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxComputerBuilt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Built</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSaveComputer">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save Computer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
